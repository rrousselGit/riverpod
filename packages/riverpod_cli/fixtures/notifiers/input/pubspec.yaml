name: codemod_riverpod_test_notifiers
description: Test riverpod codemods
version: 0.0.1
publish_to: none

environment:
  sdk: ">=2.12.0-0 <3.0.0"

dependencies:
  flutter:
    sdk: flutter
  flutter_hooks: any
  flutter_riverpod: ^0.13.0
  hooks_riverpod: ^0.13.0
  riverpod: ^0.13.0
