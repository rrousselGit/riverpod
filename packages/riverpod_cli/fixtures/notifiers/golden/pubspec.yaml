name: codemod_riverpod_test_notifiers_golden
description: Test riverpod codemods
version: 0.0.1
publish_to: none

environment:
  sdk: ">=2.12.0-0 <3.0.0"

dependencies:
  flutter:
    sdk: flutter
  flutter_hooks:
  flutter_riverpod: ^0.14.0
  hooks_riverpod: ^0.14.0
  riverpod: ^0.14.0
